examples/update-sshkey-secret

```diff
@@ -61,15 +55,14 @@ fi
 vault token-lookup > /dev/null 2>&1
 authenticate
 TMPDIR=$(mktemp -d -u)
-mkdir -p $TMPDIR
-cd $TMPDIR
+mkdir -p "$TMPDIR"
+
+if ssh-keygen -f "$TMPDIR"/private_ssh_key -N ''; then
+  base64 "${TMPDIR}/private_ssh_key" \
```

Comment:
Same thing here, no need to base64 encode the secret since it's no longer required to process the secrets